x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/configure_pipeline.tsx

```diff
@@ -0,0 +1,212 @@
+                  { defaultMessage: 'Select a model' }
+                ),
+                value: '',
+              },
+              ...models.map((m) => ({ text: m.model_id, value: m.model_id })),
+            ]}
+            onChange={(e) =>
+              setInferencePipelineConfiguration({
+                ...configuration,
+                modelID: e.target.value,
+              })
+            }
+          />
+        </EuiFormRow>
+        <EuiSpacer />
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiFormRow
+              label={i18n.translate(
+                'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.configure.sourceFieldLabel',
+                {
+                  defaultMessage: 'Source field',
+                }
+              )}
+            >
+              <EuiSelect
+                value={sourceField}
+                options={[
+                  {
+                    disabled: true,
+                    text: i18n.translate(
+                      'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.configure.sourceField.placeholder',
+                      { defaultMessage: 'Select a schema field' }
+                    ),
+                    value: '',
+                  },
+                  ...(sourceFields?.map((field) => ({
+                    text: field,
+                    value: field,
+                  })) ?? []),
+                ]}
+                onChange={(e) =>
+                  setInferencePipelineConfiguration({
+                    ...configuration,
+                    sourceField: e.target.value,
+                  })
+                }
+              />
+            </EuiFormRow>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiFormRow
+              label={i18n.translate(
+                'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.configure.destinationField.label',
+                {
+                  defaultMessage: 'Destination field (optional)',
+                }
+              )}
+              helpText={
+                formErrors.destinationField === undefined &&
+                i18n.translate(
+                  'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.configure.destinationField.helpText',
+                  { defaultMessage: 'Your field name will be prefixed with "ml.inference."' }
+                )
+              }
+              error={formErrors.destinationField}
+              isInvalid={formErrors.destinationField !== undefined}
+            >
+              <EuiFieldText
+                placeholder={i18n.translate(
+                  'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.configure.destinationField.placeholder',
+                  {
+                    defaultMessage: 'custom_field_name',
```

Comment:
I don't know if this needs to be translated to be honest, since its meant as a placeholder for a value